src/tests/texts/KeyValueUpdaterTest.java

```diff
@@ -0,0 +1,39 @@
+import org.junit.Before;
+import org.junit.Test;
+import texts.KeyValueUpdater;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertFalse;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class KeyValueUpdaterTest {
+    private KeyValueUpdater textUpdate;
+
+    @Before
+    public void setup() {
+        textUpdate = new KeyValueUpdater();
+    }
+
+    @Test
+    public void TestItAddsAKeyAndValueWhenNoExisitingText() {
```

Comment:
I would remove the prefix `Test` to your test names. They don't really add anything to the description of the tests and they break the flow when reading the test names to understand the behavior of the class. "Existing" is also misspelled.